examples/Misc/Program.cs

```diff
@@ -34,7 +34,7 @@ static void ListGroups(string brokerList)
 
             using (var producer = new Producer(config))
             {
-                var groups = producer.ListGroups(TimeSpan.FromSeconds(10));
+                var groups = producer.ClientInstance.ListGroups(TimeSpan.FromSeconds(10));
```

Comment:
I agree this approach is worse than what we have, but I just wanted to have something related to this up for general comment (will close this PR out at some point). Maybe there is some other way of thinking about this.

With re-sharper enabled (not sure if not), VS shows direct methods of a class in bold in intellisense and methods on base classes in normal weight. So deriving from a common base class to get these methods does go some way towards helping.

Note though that `Consumer<TKey,TValue>` doesn't derive from `Consumer` (because otherwise it would expose the `byte[]` methods which is not desirable. Same for `Producer`.